website/docs/d/storage_account_sas.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_storage_account_sas"
+sidebar_current: "docs-azurerm-datasource-storage-account-sas"
+description: |-
+  Create a Shared Access Signature (SAS) for an Azure Storage Account.
+
+---
+
+# Data Source: azurerm_storage_account_sas
```

Comment:
🤔 given `sas is storage account secret, it is worth naming this `azurerm_storage_account_secret/token`?

Reply:
For posterity: SAS is `shared access signature` in the context of this data source. It's not actually even really a signature by itself, but a whole URL query string, which is how the Azure tools and even Azure storage explorer use these.